cnf-tests/testsuites/validationsuite/cluster/validation.go

```diff
@@ -40,6 +40,77 @@ func init() {
 	machineConfigPoolNodeSelector = fmt.Sprintf("node-role.kubernetes.io/%s", roleWorkerCNF)
 }
 
+type MCMatcher func(*igntypes.Config, *clientmachineconfigv1.MachineConfig) bool
```

Comment:
Saw both the matcher rely only on ignitionConfig, I'd remove the second parameter as nobody uses it

Reply:
That's a future-facing hint to the matcher I'm about to add for my other PR, but it's true I could remove this parameter for now and re-add it in that PR.

Reply:
Done.